corehq/apps/app_manager/views/forms.py

```diff
@@ -199,12 +199,13 @@ def to_dict(properties):
 
 @no_conflict_require_POST
 @login_or_digest
-@require_permission(Permissions.edit_apps, login_decorator=None)
 def edit_form_attr(request, domain, app_id, unique_form_id, attr):
     """
     Called to edit any (supported) form attribute, given by attr
 
     """
+    if not request.couch_user.has_permission(domain, Permissions.edit_apps.name):
+        return HttpResponse(_("You don't have permissions to edit this"), status=403)
```

Comment:
It would be nice to be able to still use the `@require_permission`. would it be possible to make this change on the front end by checking the status code of  the response? or alternatively, have a parameter for `require_permission` or separate decorator (e.g. `require_permission_api`) that returns a more API-friendly response in text instead of the full blown page.